compiler/env/AOTAdapter.hpp

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
```

Comment:
the commit abstract seems like it was cut off mid-edit...

Reply:
also, please case correct AOT (be consistent AOT not AoT or aot).

Reply:
Will fix that, thank you!